covid19_sfbayarea/data/marin.py

```diff
@@ -27,115 +92,72 @@ def get_county() -> Dict:
-    # Some metadata strings are repeated, so use sets to dedupe them.
-    metadata: set = set()
-    chart_metadata: set = set()
 
-    with get_firefox() as driver:
-        driver.implicitly_wait(30)
-        driver.get(url)
-        soup = BeautifulSoup(driver.page_source, 'html5lib')
-
-        for soup_obj in soup.findAll('div', attrs={"class":"surveillance-data-text"}):
-            if soup_obj.findAll('p'):
-                metadata = set({paragraph.text.replace("\u2014","").replace("\u00a0", "").replace("\u2019","") for paragraph in soup_obj.findAll('p')})
-            else:
-                raise ValueError('Metadata location has changed.')
-
-    with get_firefox() as driver: # I keep getting a connection error so maybe I need to do this again? seems weird.
-        driver.implicitly_wait(30)
-        driver.get(url)
-        # Metadata for each chart visualizing the data of the csv file I'll pull.
-        # I had to change my metadata function b/c for whatever reason, my usual code didn't pick up on the class notes block.
-        # There's something weird with the website that Ricardo and I couldn't quite pinpoint.
-        source_list: set = set()
-        for chart_id in chart_ids.values():
-            driver.implicitly_wait(30)
-            source = driver.find_element_by_css_selector(f'iframe[src*="//datawrapper.dwcdn.net/{chart_id}/"]').get_attribute('src')
-            source_list.add(source)
-
-    with get_firefox() as driver:
-        for source in source_list:
-            driver.get(source)
-            #breakpoint()
-            time.sleep(5) # this ensures there's enough time for the soup to find the elements and for the chart_metadata to populate.
-            # From the source code it seems that .get() should be synchronous but it's not working like that :(
-            soup = BeautifulSoup(driver.page_source, 'html5lib')
-            for data in soup.findAll('div', attrs = {'class': 'notes-block'}):
-                #breakpoint()
-                chart_metadata.add(data.text.strip())
+def get_chart_meta(page: MarinDashboardPage, chart_ids: Iterable[str]) -> str:
+    """
+    Get all the metadata underneath the data wrapper charts and the metadata at
+    the top of the county dashboard.
+    """
+    metadata: List[str] = []
+    chart_metadata: List[str] = []
+
+    paragraphs = page.find_all('div.surveillance-data-text p')
+    if paragraphs:
+        # TODO: it's not clear why any of these are being removed, nor
+        # why they are not being replaced with an equivalent ASCII
+        # character or just a space (not having something else in their
+        # place results in joined up words, like "arealways")
+        # \u2014 = em dash
+        # \u00a0 = non-breaking space
+        # \u2019 = apostrophe/right single quote
+        metadata.extend(paragraph.text.replace("\u2014","").replace("\u00a0", "").replace("\u2019","")
+                        for paragraph in paragraphs)
```

Comment:
I *think* the right thing here is to leave the em-dashes and apostrophes in, but replace the non-breaking space with a normal space (although even that is debatable — in the news scraper, we generally err on the cautious side of not changing them in case it really is important that two words stay together).

Would love any feedback on this.